sputnikdao2/src/lib.rs

```diff
@@ -400,4 +465,67 @@ mod tests {
             },
         });
     }
+
+    #[test]
+    fn test_action_log() {
+        let mut context = VMContextBuilder::new();
+        testing_env!(context.predecessor_account_id(accounts(0)).build());
+        let accounts_list: Vec<AccountId> = [
+            "alice", "bob", "charlie", "danny", "eugene", "fargo", "grace", "hannah", "ian",
+            "julia", "kevin", "linda", "michael", "nathan", "olivia", "patricia", "quinn",
+            "robert", "sarah", "thomas", "ursula", "victor", "wendy", "xavier", "yasmine", "zack",
+            "adam", "bella", "cameron", "diana", "ethan", "fiona", "george", "heidi", "isaac",
+            "jennifer", "kyle", "lily", "marcus", "nina", "oscar", "penelope", "quincy", "ryan",
+            "sophia", "tyler", "uma", "vincent", "willow", "xander",
+        ]
+        .iter()
+        .map(|name| name.parse().unwrap())
+        .collect();
+        let mut contract = Contract::new(
+            Config::test_config(),
+            VersionedPolicy::Default(accounts_list.clone()),
+        );
+        let id = create_proposal(&mut context, &mut contract);
+
+        // Check that last actions log has proposal create
+        let proposal = contract.get_proposal(id).proposal.latest_version();
+        let proposal_kind = contract.get_proposal(id).proposal.latest_version_ref().kind;
+        let proposal_last_actions_log = proposal.last_actions_log;
+        let global_last_actions_log = contract.get_actions_log();
+        assert_eq!(proposal_last_actions_log.len(), 1);
+        assert_eq!(global_last_actions_log.len(), 1);
+        assert_eq!(
+            proposal_last_actions_log.get(0).unwrap().clone(),
+            BlockLog { block_height: 0 }
+        );
```

Comment:
Let's set the block_height in the VMContextBuilder to some realistic block height: https://docs.rs/near-sdk/latest/near_sdk/test_utils/struct.VMContextBuilder.html#method.block_height